angular/core/services/mention_service.coffee

```diff
@@ -0,0 +1,14 @@
+angular.module('loomioApp').factory 'MentionService', (Records, Session) ->
+  new class MentionService
+    applyMentions: (scope, model) ->
+      scope.updateMentionables = (fragment) ->
```

Comment:
Turns out the code for mentions works everywhere; this is pasted straight from the existing code.